tests/plots/test_scatter.py

```diff
@@ -96,3 +97,15 @@ def test_scatter_categorical(categorical_explanation):
     shap.plots.scatter(categorical_explanation[:, "sex"], ax=ax, show=False)
     plt.tight_layout()
     return fig
+
+
+@pytest.mark.parametrize("input", [np.array([[1], [1]]), np.array([[1e-10], [1e-9]]), np.array([[1]])])
+def test_scatter_plot_value_input(input):
+    """Test scatter plot with different input values."""
```

Comment:
looks good. Two final things: would you be so kind and refer to the issue as well:

GH #4037

So that we can easily check where this is originating from.

Secondly, could you pls return the figure, like it's done in other tests and add the 
`@pytest.mark.mpl_image_compare` decorator and run

```bash
pytest test/plots/test_scatter.py::test_scatter_plot_value_input --mpl-generate-path=tests/plots/baseline
```
this will then add a file with the name of the test to `tests/plots/baseline` and then add this to git as well? This will help us keep the plots consistent across releases.